### DISCUSSION TITLE
how to show all the content about a matrix with clion?

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#202] [Q&A]
Right now it is only showing the first element of the array but I want a visual of all the elements in the array. how to do it?  I tried some ways from the web, but is donot work. Looking for help. Thanks.

**Comment by tmayerguerr** [comment]
Dear zdwangjian5

Unfortunately, we cannot provide support for specific IDEs. We do our best to answer all questions, but we are only a small research group and are busy with the usual work of teaching, research and project work.

regards
Torsten
